target/imagebuilder/files/Makefile

```diff
@@ -45,6 +45,14 @@ Building images:
 	make image FILES="<path>" # include extra files from <path>
 	make image BIN_DIR="<path>" # alternative output directory for the images
 	make image EXTRA_IMAGE_NAME="<string>" # Add this to the output image filename (sanitized)
+
+Print manifest:
+	List "all" packages which get installed into the image.
```

Comment:
What is expressed by the quotes around `"all"`? I don't get it.

Reply:
Copied from the original PR, I can remove that. @feckert can you elaborate?

Reply:
If you build a new image with imagebuilder and add a package to PACKAGES variable then this package could also have dependency which also get installed. With this sentence i want to explain that the output shows all packages not only the content of the PACKAGES variable.